### DISCUSSION TITLE
Default 404 pages in Nuxt

### DISCUSSION THREAD

**Discussion by lightningnetworkstores** [nuxt/nuxt#9444] [Questions (Nuxt 2)]
When doing SSR two years ago, it seems like there was [no easy way](https://stackoverflow.com/questions/58325686/how-to-handle-nuxt-ssr-errors-and-show-custom-404-500-pages) of providing a default `error.vue` page. It's still not working with me...

Or am I missing something and there is some easy trick/workaround?

**Comment by TheAlexLichter** [comment]
It depends on what you want. A default *404 page* is not a problem at all via `errors.vue`. Uncaught errors on the server side are another topic though. When an error isn't caught on the server-side, Nuxt will stop early and show the famous gray error page as it cannot proceed to start the actual Nuxt application, boot up Vue and so on.

This error page [can be changed](https://blog.lichter.io/posts/nuxtjs-change-server-error-page/) but you cannot utilize components in there due to the fact that Nuxt had to interrupt the usual lifecycle. The easiest workaround is actually *catching* those errors and redirect to the error page. Nevertheless, providing such an error page by using HTML (and - if you want - Vue via CDN) is a good practice.

Hope that helps 👍

**Comment by pwango** [comment]
we need a easy method the solve this problem
